requirements/rocm-test.txt

```diff
@@ -20,6 +20,7 @@ librosa==0.10.2.post1
 # Entrypoints test
 #vllm[video] # required by entrypoints/openai/test_video.py
 decord==0.6.0
+rapidfuzz==3.12.1
```

Comment:
Why do we only want to have this dependency for rocm? This test will also be run with nvidia.